src/plugins/badge.coffee

```diff
@@ -584,6 +584,8 @@ class Badge
 
     stars
 
+  _displayStars: (rating) -> rating isnt 0
```

Comment:
Consider renaming `_displayStars` to `_noStars` (returning the opposite value):

```coffeescript
_noStars: (rating) -> rating is 0
```
as the following snippets become more appropriate: 

```coffeescript
 _ratingToStars: (rating = 0, limit = 5) ->
  return [] if @_noStars(rating)
  ..
```

```coffeescript
..
@$embeddedBadgePlugin().className += ' sa-badge-no-stars' if @_noStars(rating)
..
```